modules/backend/formwidgets/mediafinder/assets/js/mediafinder.js

```diff
@@ -95,7 +99,8 @@
 
         new $.wn.mediaManager.popup({
             alias: 'ocmediamanager',
-            cropAndInsertButton: true,
+            cropAndInsertButton: ["image", "all"].indexOf(self.options.mode) > -1,
```

Comment:
We can use ES6 code since we don't support IE11 anymore ;)
```suggestion
            cropAndInsertButton: ['image', 'all'].includes(self.options.mode),
```